README.md

```diff
@@ -93,34 +62,43 @@ try {
 }
 ```
 
-> Exploring not requiring JS and enabling this functionality declaratively by the page having a hidden field that the browser will fill with the SD-JWT that gets posted to the RP server.
+### Aggregation
 
+On page load, the browser:
 
-2. User clicks in input field and browser displays selection of what could be shared. Emails that would be verified are decorated for user to understand. User selects a verified email from browser selection.
+1. Loops through the `configURL`s that were registered previously and stored in browser storage in the previous step to build a list of suggestions:
 
-3. The `navigator.credentials.get()` call returns and `credential.token` is an SD-JWT+KB
+#### For each registered provider
 
-``` 
-// token example and payload
-```
-
-5. JS code sends `token` to RP server. 
+1. The browser fetches the well-known file for `https://issuer.example.com` from `https://issuer.example.com/.well-known/web-identity` per https://w3c-fedid.github.io/FedCM/#idp-api-well-known
```

Comment:
How does the browser know to fetch this from issuer.example.com, when the registered provider was on issuer.example.net? Was this just a typo? If so, then what does the /fedcm.json accomplish during the registration call? Is the issuer registering a full URL or an origin?

Reply:
> Was this just a typo?

I think this was just a typo.

> If so, then what does the /fedcm.json accomplish during the registration call? 

The browser discovers the IdP's endpoints and configuration (described [here](https://w3c-fedid.github.io/FedCM/#config-file)).

For this specific feature, it needs to discover things like the "vc_issuance_endpoint".

> Is the issuer registering a full URL or an origin?

It is registering a URL where the browser can find the [`config`](https://w3c-fedid.github.io/FedCM/#config-file) file.